### PR TITLE
Redirect correctly when URL matches multiple rows

### PR DESCRIPTION
**Pull request by louy2**
We find the row in infobox with URL by regex matching on URL or Website.
This commit handles the edge case that some infoboxes mentions URL or
Website before the actual URL row.
The test case is recorded in a new testcases.txt file.

### CODE REVIEW

File: browser-extension/common.js [hunk 1]
```diff
@@ -17,7 +17,13 @@ function redirect(searchQuery) {
   var wikiUrlRow = infoboxRows.filter(x => x.innerText.match(/(?:URL)|(?:Website)/));
 
   if (wikiUrlRow[0]) {
-    return wikiUrlRow[0].querySelector('a').href;
+    for (var i in wikiUrlRow) {
+      var a = wikiUrlRow[i].querySelector('a');
```
Comment (ajitzero) [inline browser-extension/common.js:21]:
```suggestion
    for (let i in wikiUrlRow) {
      let a = wikiUrlRow[i].querySelector('a');
```
